### ISSUE TITLE
OpenStack rate limit validation doesn't reveal potential problems

### ISSUE THREAD

**Issue by loewenstein** [cloudfoundry/docs-deploying-cf#105]
As part of our efforts to automate OpenStack validation (https://github.com/cloudfoundry-incubator/cf-openstack-validator) we introduce a rate limit test.

We just discovered two things:
* Fog caches the results of `compute.servers`, i.e. a valid check would be `servers = compute.servers; 100.times { servers.reload }`
* GET is in general not limited, i.e. just enabling rate limits with default values wouldn't limit the calls to list servers

https://github.com/cloudfoundry/docs-deploying-cf/blob/master/openstack/validate_openstack.html.md.erb#L36-L41

What is the background of this specific check?

**Comment by ljarzynski** [comment]
Hi @loewenstein! I just saw this slack thread and wanted to follow up with you https://cloudfoundry.slack.com/archives/cf-docs/p1465928312000007. Do you have any updates surrounding this issue?

**Comment by loewenstein** [comment]
No. Unfortunately I did not get any response yet.

**Comment by ljarzynski** [comment]
Hello @loewenstein, if you haven't heard anything back, I think I'm going to go ahead an close this issue. But please feel free to re-open if you get any new information!

**Comment by loewenstein** [comment]
Hi @ljarzynski, 

The current description in the docu is apparently pretty useless (though harmless as well). 
1. The code snippet executes the REST call once instead of a 100 times (at least for recent versions of fog). 
2. Recent versions of OpenStack have no rate limit for GET requests and a default of 120/minute for other requests. 
Thus I would remove (or at least adapt) the docu unless you can trace down the origin of that requirement and figure out that it indeed still applies. 

My 2 cents. 

Regards 
Jan

**Comment by ljarzynski** [comment]
@loewenstein That is helpful! I'll figure out who I can talk to about this.

**Comment by ljarzynski** [comment]
@loewenstein mentioned you in the tracker story: https://www.pivotaltracker.com/story/show/121205093

**Comment by ljarzynski** [comment]
Closing this issue as we received the following explanation:

"I don't know how we came to that example to validate the API rate limit. It never make sense, even on previous OpenStack versions, the default rate limit for GET request was higher than 100req/sec, it only makes sense to validate POST request. I remember that we were hit frequently when updating the server metadata, so maybe this is a better validation." 

Docs will be updated via PR.
